sqld/src/utils/services/idle_shutdown.rs

```diff
@@ -37,6 +37,10 @@ impl IdleShutdownLayer {
             watcher: Arc::new(sender),
         }
     }
+
+    pub fn kick(&self) {
+        let _: Result<_, _> = self.watcher.send(());
```

Comment:
Why is it ok to ignore this result?

Reply:
It seems that `let _ = self.watcher.send(());` should be enough, no?

Reply:
Yes, it would be enough, but the explicit type makes sure I actually drop the type that I want. It has happened to me that I wrote `let _ = channel.send(something);` where `channel` was a mpsc channel where the `send()` method is async, so instead of sending an item on the channel and ignoring the result, I ignored the future! This taught me to specify the ignored type.

By a funny coincidence, I made the same mistake today, and the compiler caught the error because I specified the type.